app/models/category.rb

```diff
@@ -10,6 +10,12 @@ class Category < ApplicationRecord
     inverse_of: :parent
   )
 
+  has_many(:visible_subcategories, -> { visible.order(:display_order, :name) },
+           class_name: "Category",
+           foreign_key: :parent_id,
+           inverse_of: :parent
+  )
+
```

Comment:
I'm curious why this needs to be a separate `has_many` instead of a `scope`?

Reply:
@exbinary I thought it needed to be an assn, per this language in the blueprint:

<img width="782" alt="Screen Shot 2020-07-24 at 10 20 47 AM" src="https://user-images.githubusercontent.com/7607813/88401270-83032a80-cd97-11ea-8edd-2760bf93a455.png">